react/tic-tac-toe/src/app/screens/Game/components/Square/index.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import styles from './styles.module.scss';
 
-const Square = () => (
-  <button type="button" className={styles.square}>
-    {/* TODO */}
-  </button>
-);
+function Square({ value, index, onClick }) {
+  return (
+    <button type="button" className={styles.square} onClick={() => onClick(index)}>
+      {value}
```

Comment:
Please avoid using an arrow function in the click handler, because every time render runs,  you are creating a new function reference, and react detects a props difference. If you have child components, they'll re-render even if data hasn't change.
You have to find a way to avoid calling this inline function 💪